app/src/main/kotlin/com/exallium/todoapp/allnotes/AllNotesPresenter.kt

```diff
@@ -25,4 +32,10 @@ class AllNotesPresenter(private val view: AllNotesView,
         adapter.cleanup()
     }
 
-}
\ No newline at end of file
+    private fun makeNoteDetailBundle(id: String?) : Bundle {
+        val args: Bundle = bundleFactory.createBundle()
+        screenBundleHelper.setNoteId(args, id)
+        screenBundleHelper.setTitle(args, R.string.note_detail_screen_title)
```

Comment:
done. we could go even further and have the view handle it (maybe setScreenTitle() on the BaseView but it won't be needed for the main screen). i think it's ok as it is. what do you think?